USWDS 3.0 - "Introducing USWDS 3.0" edits (#1561)
uswds/uswds-site: _posts/2022-04-28-introducing-uswds-3-0.md

### Update to modern Sass syntax 

USWDS 3.0 stylesheets use the most modern version of the Sass module syntax. The Sass language that we use to write our stylesheets updated its syntax in 2020 and the old syntax (using `@import`) is now deprecated. Sass will no longer support the old syntax starting in October 2022. We want teams to use source code with reliable support throughout the life of their project, and USWDS 3.0 allows teams to migrate to Sass modules and maintain support.

[Read more about the Sass module syntax](https://sass-lang.com/)

### Improve performance and reduce the size of your project CSS 

Unbundle the design system with USWDS 3.0. Using USWDS 3.0 with Sass module syntax allows teams to unbundle their implementations of USWDS and use only the components and code that they need on their project. Depending on your project, this could mean significant reduction of CSS code you ship. This means faster load times, better scores in performance evaluation tools, and a better developer experience. 

> [Read more about USWDS Packages](/components/packages/) →

Reviewer: ```suggestion
[Read more about USWDS Packages]({{ site.baseurl }}/components/packages/) →
```

Author: Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)

> [USWDS browser support (2% rule)](https://designsystem.digital.gov/documentation/developers/#browser-support) →

Reviewer: ```suggestion
[USWDS browser support (2% rule)]({{ site.baseurl }}/documentation/developers/#browser-support) →
```

Author: Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)

[See current analytics from analytics.usa.gov](https://analytics.usa.gov/)

### Support some Javascript-required functionality 

We believe that JavaScript-disabled environments fall below our 2% support threshold, and that we may require JavaScript for some design system functionality. While we have not made any changes to our codebase that affect how we use JavaScript with our components, we may make changes that require JavaScript for some functionality as we move forward. When we do make any changes like this, we'll be sure to note it in our release notes.

### Support USWDS 2 with maintenance and security patches for one year

> In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/discussions/4626).

Reviewer: ```suggestion
In addition to supporting active development on USWDS 3, we will continue to support the USWDS 2 branch for one year of maintenance and security patches (until April 2023). The current LTS version of USWDS 2 is [USWDS 2.13.3](https://github.com/uswds/uswds/releases/tag/v2.13.3).
```

Let's link to the release, not the discussion

Author: Updated in [769f8c3](https://github.com/uswds/uswds-site/pull/1561/commits/769f8c3ac529340c57d075dcaf71a70af01e120c)

> [Long-term support of version 2.x]({{ site.baseurl }}/documentation/developers/#long-term-support-of-v2x) → 

Author: @thisisdano
I am not able to find the content for this link

Reviewer: It should be there — possibly related to caching of the old README?

<img width="775" alt="Screen Shot 2022-04-27 at 3 38 39 PM" src="https://user-images.githubusercontent.com/11464021/165642885-f9e15087-e411-4441-98ba-08569654a76e.png">

Author: My bad. I cleared the ole jekyll cache and all was well. Marking this as resolved.
